### PR TITLE
As a user, I want to have the PXE format accessible from Hammr

### PR DESCRIPTION
**Pull request by CanutMIAGE**
…Hammr

-Added possibility to generate PXE images
-Added tests

### CODE REVIEW

File: documentation/en/source/pages/templates-spec/builders/builders-pxe.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Default builder type: ``PXE``
+
+Require Cloud Account: No
+
+This builder type is the default name provided by UForge AppCenter.
+
+.. note:: This builder type name can be changed by your UForge administrator. To get the available builder types, please refer to :ref:`command-line-format`
+
+The PXE builder provides information for building PXE images.
```
Comment (cinlloc) [inline documentation/en/source/pages/templates-spec/builders/builders-pxe.rst]: You should explain in one more sentence how PXE image generated can be used (having a PXE boot, etc)

File: documentation/en/source/pages/templates-spec/builders/builders-pxe.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.. Copyright (c) 2007-2016 UShareSoft, All rights reserved
```
Comment (jgweir) [inline documentation/en/source/pages/templates-spec/builders/builders-pxe.rst]: Just a small request....could we update the date on the copyright to 2017.
Thanks

Comment (CanutMIAGE) [inline documentation/en/source/pages/templates-spec/builders/builders-pxe.rst]: A bug will be created on Jira in order to change update every copyrights. Would that be ok for you?

File: tests/integration/data/templatePXE.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "os" : {
+      "name" : "CentOS",
+      "version" : "7",
+      "arch" : "x86_64",
+      "profile" : "Minimal"
+    }
+ },
+ "builders" : [
+ {
+  "type" : "KVM",
```
Comment (cinlloc) [inline tests/integration/data/templatePXE.json]: The type should be "PXE", shouldn't it?

File: tests/integration/test.py [hunk 4]
```diff
@@ -159,6 +159,12 @@ def test_08_1_exportCentOS6(self):
                 os.remove("archive.tar.gz")
                 self.assertEqual(r, 0)
 
+        def test_09_buildPXE(self):
+                template = hammr.commands.template.Template()
+                template.set_globals(api, login, password)
+                r = template.do_build("--file data/template.json")
```
Comment (cinlloc) [inline tests/integration/test.py]: Should be `templatePXE.json`

File: documentation/en/source/pages/templates-spec/builders/builders-pxe.rst (hunk 2 above)
Comment (segalaj) [inline documentation/en/source/pages/templates-spec/builders/builders-pxe.rst]: I agree with @jgweir comment, please change the copyright here to 2017, and you could open an issue for old files

File: tests/integration/data/templatePXE.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      "name" : "CentOS",
+      "version" : "7",
+      "arch" : "x86_64",
+      "profile" : "Minimal"
+    }
+ },
+ "builders" : [
+ {
+  "type" : "PXE"
+ }
```
Comment (segalaj) [inline tests/integration/data/templatePXE.json]: opt: could be idented

**Pull request merged**